content/client-lib-development-guide/features.textile

```diff
@@ -287,9 +292,10 @@ h3(#realtime-connection). Connection
 * @(RTN6)@ A @Connection@ is successful and considered @CONNECTED@ once the websocket connection is open and the intial @CONNECTED@ @ProtocolMessage@ has been received
 * @(RTN19)@ If the @CONNECTED@ @ProtocolMessage@ contains a @connectionDetails@ property, the attributes within @ConnectionDetails@ will be used as the defaults for this client library, overriding any configured options at the time the @CONNECTED@ @ProtocolMessage@ is received
 * @(RTN7)@ @ACK@ and @NACK@:
-** @(RTN7a)@ All @ProtocolMessage@ @Presence@ and @Message@ objects sent to Ably expect either an @ACK@ or @NACK@ from Ably to confirm successful receipt and acceptance or failure respectively
+** @(RTN7a)@ All @ProtocolMessage@ @Presence@ and @Message@ objects sent to Ably expect either an @ACK@ or @NACK@ from Ably to confirm successful receipt and acceptance or failure respectively. For clarity, it is unnecessary to fail the publish operation of a message using a timer. Instead the client library can rely on the realtime system to send an @ACK@ or @NACK@ when connected, to fail all awaiting messages when @DISCONNECTED@ for too long as described in "RTN7d":#RTN7d, or upon reconnecting, the corresponding @ACK@ or @NACK@ protocol messages will be received as described in "RTN19a":#RTN19a
```

Comment:
Well I referenced RTN19a so that people understand why the ACK or NACK would be received, but clearly the wording is not clear enough so I'll improve.